components/nexus-repository/src/main/java/org/sonatype/nexus/repository/config/ConfigurationFacetImpl.java

```diff
@@ -42,6 +43,9 @@
     extends FacetSupport
     implements ConfigurationFacet
 {
+  private static final TypeReference<Map<String, Object>> MAP_STRING_OBJECT =
+      new TypeReference<Map<String, Object>>() { };
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)